win32ss/pch.h

```diff
@@ -74,6 +74,12 @@ typedef struct _SECURITY_ATTRIBUTES SECURITY_ATTRIBUTES, *LPSECURITY_ATTRIBUTES;
 /* SEH support with PSEH */
 #include <pseh/pseh2.h>
 
+/* The native x64 definition of FLOATOBJ_GetFloat in winddi.h is retarded. Use this instead. */
```

Comment:
Seriously retarded indeed... Maybe explicit why it's so retarded in the comment. I had to look it up to see for myself.